internal/hud/renderer.go

```diff
@@ -163,19 +176,17 @@ func (r *Renderer) renderResourceLogModal(res view.Resource, background rty.Comp
 		s = fmt.Sprintf("No log output for %s", res.Name)
 	}
 
-	return r.renderLogModal(fmt.Sprintf("%s pod log", res.Name), s, background)
+	return r.renderLogModal(fmt.Sprintf(" POD LOG: %s ", res.Name), s, background)
 }
 
 func (r *Renderer) renderLogModal(title string, s string, background rty.Component) rty.Component {
 	sl := rty.NewTextScrollLayout(logScrollerName)
 	sl.Add(rty.TextString(s))
 	box := rty.NewBox()
 	box.SetInner(sl)
-	box.SetTitle(title)
+	box.SetTitle(fmt.Sprintf(" %s ", title))
```

Comment:
oh cool, I wanted to do this, but I think it'd be better to do in the box implementation itself

Reply:
I had the same thought! I spent a minute digging into how to make the change in RTY but it wasn't obvious, so let's look at this together.